config/nunjucks.config.js

```diff
@@ -49,6 +49,14 @@ function configure(env) {
     return moment(date).format('HH:mm');
   });
 
+  env.addFilter('nicedatetime', (date) => {
```

Comment:
I think the GO.UK styleguide is like 4 June 2017: https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#dates

Reply:
noice